app/src/main/java/org/literacyapp/authentication/AuthenticationActivity.java

```diff
@@ -149,23 +189,26 @@ public Mat onCameraFrame(CameraBridgeViewBase.CvCameraViewFrame inputFrame) {
     public void onResume()
     {
         super.onResume();
-        TrainingHelper trainingHelper = new TrainingHelper(getApplicationContext());
-        svm = trainingHelper.getSvm();
-        tensorFlow = trainingHelper.getInitializedTensorFlow();
         ppF = new PreProcessorFactory(getApplicationContext());
         numberOfTries = 0;
         animalOverlay = animalOverlayHelper.createOverlay();
-        if (animalOverlay != null){
+        if (animalOverlay != null) {
             mediaPlayerAnimalSound = MediaPlayer.create(this, getResources().getIdentifier(animalOverlay.getSoundFile(), "raw", getPackageName()));
-            mediaPlayerInstruction.setOnCompletionListener(new MediaPlayer.OnCompletionListener() {
+        }
+        preview.enableView();
```

Comment:
This is the OpenCV camera preview which needs to be enabled (this start the whole thread which gives us the camera frames)